index.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const pkg = require('./package.json');
+const Hoek = require('hoek');
+
+const defaults = {
+  parserOptions: {
```

Comment:
can we add the pagedata site, slug, tag to this option so we don't have to pass in every time?